main.php

```diff
@@ -12,7 +12,7 @@ function truncate($body, $truncate) {
 	}
 }
 
-function fetch_translation(&$page, $lang, $show_fallback_text=true)
+function add_content(&$page, $lang)
```

Comment:
could you please split the removal of `$show_fallback_text` into a separate commit?

Reply:
I was wondering where this even came from and I found it: 402a21bf44f23e1ac50c3267932edec1efeaca50